### PR TITLE
OIE e2e Scenario 6.1.3 - Mary enters a wrong verification code

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
```
  Scenario 6.1.3: Mary enters a wrong verification code
	GIVEN Mary navigates to the Basic Login View
	WHEN she fills in her username
	AND she fills in her correct password
	AND she clicks Login
	THEN She sees a list of factors
	WHEN She has selected Email from the list of factors
	AND She inputs the incorrect code from the email
	THEN the sample shows an error message "Invalid code. Try again." on the Sample App
```

[OKTA-397483](https://oktainc.atlassian.net/browse/OKTA-397483)

### CODE REVIEW

File: samples/test/steps/given.ts [hunk 1]
```diff
@@ -40,6 +45,11 @@ Given(
   navigateToLoginAndAuthenticate
 );
 
+Given(
+  /^a User named "([^/s]+)" created in the admin interface with a Password only$/,
+  _ => ({}) // todo: createAndStoreUserInContext
```
Comment (denysoblohin-okta) [inline samples/test/steps/given.ts:50]: todo: use code in [this PR](https://github.com/okta/okta-auth-js/pull/782/files#diff-0ba77150dd83ad840eafe3f8afad07d6b889f3dede7b679df363a391e7470387R3)

File: samples/test/steps/given.ts [hunk 2]
```diff
@@ -45,7 +50,10 @@ Given(
 
 Given(
   /^([^/s]+) is a user with a verified email and a set password$/,
-  createAndStoreUserInContext
+  async function(firstName: string) {
```
Comment (shuowu) [inline samples/test/steps/given.ts:53]: why is this needed? different context (this)? if that's the case, why not just `createAndStoreUserInContext.bind(this)`?

File: samples/test/support/wait/waitForDisplayed.ts [hunk 3]
```diff
@@ -15,9 +15,24 @@ export default async (selector: Selector, falseCase: any = false) => {
     const ms = 10000;
 
     const el = await $(selector);
-    await el.waitForDisplayed({
-        timeout: ms,
-        reverse: Boolean(falseCase),
-    });
+
```
Comment (shuowu) [inline samples/test/support/wait/waitForDisplayed.ts:18]: what issue this change is trying to resolve?

Comment (denysoblohin-okta) [reply samples/test/support/wait/waitForDisplayed.ts:18]: I had problems with trying to figure out why `waitForDisplayed` fails with timeout error in CI (expected element was not found by selector on page after 10s). To debug this I return page HTML to console for investigation.  In this case problem was with credentials and I've found `Authentication failed` in HTML

File: samples/test/steps/given.ts (hunk 2 above)
Comment (denysoblohin-okta) [reply samples/test/steps/given.ts:53]: I've added 2nd param `assignToGroup?: string` to `createAndStoreUserInContext ` function.
And Cucumber supports only 1 parameter in callback function

Comment (shuowu-okta) [reply samples/test/steps/given.ts:53]: I see, looks like you can use `curry` for in that case. like `foo(arg1)(arg2)`. But not needed, there is no issue with current implementation.

File: samples/test/support/wait/waitForDisplayed.ts (hunk 3 above)
Comment (shuowu-okta) [reply samples/test/support/wait/waitForDisplayed.ts:18]: I would suggest to keep the original function unchanged. And for debugging, you probably can start with the sample app in the browser.

Comment (denysoblohin-okta) [reply samples/test/support/wait/waitForDisplayed.ts]: Reverted.

Comment (denysoblohin-okta) [reply samples/test/support/wait/waitForDisplayed.ts]: Reverted.

**Review by shuowu — APPROVED** [review]
